lib/active_stash/railtie.rb

```diff
@@ -1,5 +1,17 @@
 module ActiveStash
   class Railtie < Rails::Railtie
+    config.active_stash = ActiveSupport::OrderedOptions.new
+
+    initializer "active_stash.configure" do |app|
+      ActiveStash.configure do |config|
+        app.config.active_stash.each { |key, value| config.send("#{key}=", value) }
```

Comment:
```suggestion
        app.config.active_stash.each { |key, value| config.public_send("#{key}=", value) }
```

Just on the off chance there's a private accessor method on the configuration.  (Ditto for two lines down)

Reply:
Good catch! Fixed: https://github.com/cipherstash/activestash/pull/15/commits/0973626b1b5864f7d63d5dad53d14f3abb1d5d5d